roles/libvirt-setup/defaults/main.yml

```diff
@@ -30,20 +33,29 @@ libvirt_default_pool_dir: "{{libvirt_images_dir}}/pool-default"
 
 external_network_cidr_v4: 192.168.125.0/24
 external_network_bridge_ip: "{{ external_network_cidr_v4|ansible.utils.nthhost(1)|default('', true) }}"
-
 external_network_dhcp_range_v4:
   - "{{ external_network_cidr_v4|ansible.utils.nthhost(200)|default('', true) }}"
   - "{{ external_network_cidr_v4|ansible.utils.nthhost(250)|default('', true) }}"
-
 external_network_static_ip_start_v4: "{{ external_network_cidr_v4|ansible.utils.nthhost(200)|default('', true) }}"
 
+external_network_cidr_v6: fd55::/64
+external_network_bridge_ip_v6: "{{ external_network_cidr_v6|ansible.utils.nthhost(1)|default('', true) }}"
+external_network_dhcp_range_v6:
+  - "{{ external_network_cidr_v6|ansible.utils.nthhost(200)|default('', true) }}"
+  - "{{ external_network_cidr_v6|ansible.utils.nthhost(250)|default('', true) }}"
```

Comment:
We could use potentially different values here, but it probably makes sense to keep IPv4 and IPv6 aligned for the dual-stack case.

Reply:
Yeah for debugging it seemed easier to just keep them the same, but users can override the values if they prefer something else